pkg/query-service/telemetry/telemetry.go

```diff
@@ -249,10 +256,20 @@ func getOutboundIP() string {
 }
 
 func (a *Telemetry) IdentifyUser(user *model.User) {
+	if user.Email == "admin@admin.com" || user.Email == "admin@signoz.cloud" {
+		return
+	}
 	a.SetCompanyDomain(user.Email)
+	a.SetUserEmail(user.Email)
```

Comment:
I don't think user events will be correct due to this. The funnels created have to be based on companyDomain and any user funnel would be broken IMO. What do you say?

Reply:
How can we sort the user metrics?

Reply:
Still looking into this.

Reply:
Added group call to identify companies.
Now funnels can be created based on company domain.